doc/src/build/install.md

```diff
@@ -7,21 +7,18 @@ dependencies.  As a prerequisite, you will need to [install
 Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
 version 1.59.0 or higher in order to build and install Sui on your machine.
 
-If you'd like to install only Sui binaries (`sui`, `wallet`,
-`sui-move`, and `rest_server`), use the following command:
+### CLIs
+
+After installing `cargo`, run:
 
 ```shell
 cargo install --git https://github.com/MystenLabs/sui.git
 ```
 
-Alternatively, clone the [Sui
-GitHub](https://github.com/MystenLabs/sui) repository and then `cargo
-install` with the repository clone:
-
-```shell
-git clone https://github.com/MystenLabs/sui.git
-cargo install --path sui/sui
-```
+This will put three binaries in your `PATH`:
```

Comment:
> * Should we rename the `wallet` binary something like `sui-wallet` @patrickkuo ?
Yes we should!

> The bigger questions are about the eventual story about how many CLI's there should be + how they should interact. I'm guessing there's already some thought going into this as part of the Gateway Service API restructuring--please fill me in or point me to the place to read about it if so!
> 
> * Right now, `wallet` and `sui-rest-server` both have spinning up a Sui network _and_ a gateway service as a subroutine. These seem like things folks might want to do without running a wallet or REST server
> * The separation between the two CLI's seems perhaps unnecessary. Maybe starting a network, a gateway service, and using a wallet all want to be subcommands of a one-stop `sui` CLI? Or they could be separate--not sure.

I have a different view on combining wallet and rest-server :) .... I think wallet should be in its own package, or even better, in it's own repo; it can become a reference implementation of application using the Sui Rust/Rest SDK(when we have one), we can also use it to help us make sure the SDK have all the functionalities a application might need. 

After the Gateway Service API restructuring, the rest-server will become rest-gateway, and will no longer spin up a network internally, it will connect and become the gateway to a local network (`./sui start`) or to testnet.

The wallet will still be able to connect to the Sui Network directly (Embedded Gateway) or via the rest-gateway

Reply:
Makes sense to me + thanks for the context!!